services/_shared/personas/walkthrough/dispatch.py

```
@@ -0,0 +1,222 @@
+            f"{_API}/repos/{_repo_path(owner, repo)}/pulls/{pull_number}/files",
+            params={"per_page": 100, "page": page},
+            headers=_headers(token), timeout=_TIMEOUT,
+        )
+        resp.raise_for_status()
+        batch = resp.json()
+        for f in batch:
+            out.append(FileStat(
+                path=str(f.get("filename", "")),
+                additions=int(f.get("additions", 0)),
+                deletions=int(f.get("deletions", 0)),
+            ))
```

Ruff (0.15.20)

[warning] Use `list.extend` to create a transformed list

(PERF401)